src/main/java/frc/robot/Robot.java

```diff
@@ -28,9 +28,9 @@
  */
 public class Robot extends TimedRobot {
 
-    private Command m_autonomousCommand;
+    private Command mAutonomousCommand;
```

Comment:
this doesn't follow our naming convention (the same is true for mRobotContainer)